plugins/node/opentelemetry-instrumentation-connect/src/internal-types.ts

```diff
@@ -14,9 +14,14 @@
  * limitations under the License.
  */
 
-import type { HandleFunction, Server } from 'connect';
+import type { HandleFunction, IncomingMessage, Server } from 'connect';
+
+export const _LAYERS_STORE_PROPERTY = '__ot_middlewares';
```

Comment:
If you use a `symbol` it would not show up as enumerable property and also no clash is possible.